templates/base.html

```diff
@@ -57,8 +57,7 @@
                     <ul class="vertical menu">
                         <li><a class="nav-color" href="{% url 'home' %}">HOME</a></li>
                         <li><a class="nav-color" href="{% url 'artists' %}">ARTISTS</a></li>
-                        <li><a class="nav-color" href="{% url 'leaderboard' %}">CHARTS</a></li>
-                        <li><a class="nav-color" href="https://blog.investperdiem.com" target="_blank">BLOG</a></li>
```

Comment:
Since it is an external link, felt like it didn't flow with the rest of the site. Didn't put too much thought into changing it, just wanted to see how it felt being on the bottom instead. I'm cool with adding it back if you think it's good to have.